macOS/DuckDuckGo/Updates/Sparkle/ReleaseNotesUserScript.swift

```diff
@@ -117,7 +117,7 @@ extension ReleaseNotesUserScript {
     @MainActor
     private func retryUpdate(params: Any, original: WKScriptMessage) async throws -> Encodable? {
         DispatchQueue.main.async { [weak self] in
-            self?.updateController.checkForUpdateSkippingRollout()
+            self?.updateController?.checkForUpdate()
```

Comment:
Flaggin all the cases I'm seeing of this, since it's an important behavioral change.